content/blog/2019/r-version-support/index.md

```diff
@@ -44,7 +44,9 @@ Tidyverse minimum R version support
 
 </div>
 
-[^1]: Add table with annual historic and projected R-version support.
+[^1]: Note that we only update the required version on package release, so you may see older versions listed in published CRAN packages.
```

Comment:
If we want to be really pedantic, we could spell this out more. I.e. we don't rush to re-release all packages after a minor version increment of R itself, but rather we bump the minimum version of R at the next obvious opportunity.

Reply:
While you were looking at this I made some more changes. What do you think of the wording now?